src/applications/disability-benefits/686/config/form.js

```diff
@@ -190,6 +193,114 @@ const formConfig = {
         }
       }
     },
+    veteranInformation: {
+      title: 'Veteran Information',
+      pages: {
+        veteranInformation: {
+          title: 'Veteran Information',
+          path: 'veteran-information',
+          uiSchema: {
+            veteranFullName: _.merge(fullNameUI, {
+              'ui:options': {
+                hideIf: formData => isVeteran(formData)
+              },
+              first: {
+                'ui:title': 'Veteran’s first name',
+                'ui:required': formData => isNotVeteran(formData)
+              },
+              middle: { 'ui:title': 'Veteran’s middle name' },
+              last: {
+                'ui:title': 'Veteran’s last name',
+                'ui:required': formData => isNotVeteran(formData)
+              },
+              suffix: { 'ui:title': 'Veteran’s suffix' }
+            }),
+            veteranSocialSecurityNumber: {
+              'ui:options': {
+                updateSchema: (form, schema) => {
+                  if (isVeteran(form)) {
+                    return _.merge(schema, {
```

Comment:
I don't think you need the `_.merge` here (and in the other similar locations), by default the forms library code will do this for you and merge the object you return with the current schema.